tests/DBTestCase.php

```diff
@@ -2,22 +2,38 @@
 
 namespace Koala\Pouch\Tests;
 
-use League\Flysystem\Config;
+use Illuminate\Foundation\Testing\RefreshDatabase;
 
 abstract class DBTestCase extends TestCase
 {
-    protected $artisan;
+    use RefreshDatabase {
+        RefreshDatabase::migrateFreshUsing as parentMigrateFreshUsing;
```

Comment:
Very cool haven't seen aliasing when using a trait before like this! Hadn't seen this `insteadOf` [operator](https://www.php.net/manual/en/language.oop5.traits.php) before either.